doc/getting-started.md

```diff
@@ -0,0 +1,132 @@
+# Getting Started
+
+## Installation
+
+For the moment, only the latest development version is available. To obtain it
+you need to perform a manual clone and build of the project.
+
+First you need a GNAT compiler. On Linux you can usually get it from your
+distribution. Otherwise, and for Windows and MacOS you can download and install
+[GNAT Community](https://www.adacore.com/download).
+
+Then clone the git repository:
+
+`git clone --recursive https://github.com/alire-project/alire.git`
+
+Note the **--recursive** flag; otherwise compilation will fail.
+
+And build `alr`:
+
+1. If running on macOS, `export OS=macOS`
+1. `cd alire`
+1. `gprbuild -j0 -p -P alr_env`
+
+The executable will be found in `bin/alr`. You can add it to your PATH:
+
+ * Linux/MacOS: `export PATH=$PATH:$PWD/bin`
+ * Windows PowerShell: `$env:path += ";$pwd/bin"`
+
+## First steps
```

Comment:
We can mention here the bash autocompletion script

Reply:
I think would go in a more advanced section.